templates/builders_user_data.tpl

```diff
@@ -28,11 +28,21 @@ echo "      Pulling Server Builder Image"
 echo "-------------------------------------------"
 sudo docker pull $BUILDER_IMAGE
 
+# Make a new docker network
+docker network create -d bridge -o "com.docker.network.bridge.name"="circle0" circle-bridge
+
+# Block traffic from build containers to the EC2 metadata API
+iptables -I FORWARD -d 169.254.169.254 -p tcp -i docker0 -j DROP
+
+# Block traffic from build containers to non-whitelisted ports on the services box
+iptables -I FORWARD -d $SERVICES_PRIVATE_IP -p tcp -i docker0 -m multiport ! --dports 80,443 -j DROP
+
```

Comment:
```
> make docker the default choice for AWS, but that we should also block access to 169.254.169.254 from builds if we do that
```
Can we explain this a little more?  What is meant by make Docker default choice?  Why are we blocking that IP address?

Reply:
Also... have we tested this?